### ISSUE TITLE
Add a keyboard shortcut for fit to page.

### ISSUE THREAD

**Issue by crocket** [YACReader/yacreader#114]
Fit to page doesn't have a keyboard shortcut. It's inconvenient.

**Comment by jeremydouglass** [comment]
@crocket -- what keyboard shortcut would you expect or want to use for this?

**Comment by crocket** [comment]
I want to be able to configure the keyboard shortcut in settings.

**Comment by jseakle** [comment]
it's really annoying that there's W for full size, but no key to go back. the best solution imo would be for W to toggle between these modes, but each mode having its own key would be fine too.

**Comment by RikisX** [comment]
Yeah, shortcut would be nice. It could be Q. So it would be near W.
